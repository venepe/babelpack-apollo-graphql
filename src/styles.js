import { injectGlobal } from 'styled-components'


export const theme = {
  layout: {
    maxWidth: `${1000 / 16}rem`,
  },
  color: {
    primary: '#5d9cec',
    secondary: '#4a89dc',
    lightGreen: '#48cfad',
    darkGreen: '#37bc9b',
    lightRed: '#ed5565',
    darkRed: '#da4453',
    white: '#fff',
    lightGrey: '#ccd1d9',
    darkGrey: '#aab2bd',
    black: '#000',
  },
};

const breakpoints = {
  xs: 340,
  sm: 550,
  md: 768,
  lg: 1000,
  xl: 1200,
  xxl: 1600,
};

export const mediaQueries = Object.keys(breakpoints).reduce((obj, size) => {
  obj[size] = `@media (min-width: ${breakpoints[size] / 16}rem)`
  return obj
}, {});

const injectGlobalStyles = () => injectGlobal`
  html,
  body,
  #__next {
    height: 100%;
  }

  html *
{
   font-family: 'Cormorant Garamond' !important;
}

  body {
    min-width: 326px;
  }
  body,
  textarea,
  input,
  button,
  label {
    font-family: 'Cormorant Garamond', 'Open Sans', sans-serif;
  }
`

export default injectGlobalStyles;
