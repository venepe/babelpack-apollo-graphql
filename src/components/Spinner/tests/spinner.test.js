import React from 'react'
import { shallow } from 'enzyme'
import { theme } from 'styles'
import 'jest-styled-components'
import { Spinner } from '../index'


describe('Spinner Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <Spinner theme={theme} />,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
