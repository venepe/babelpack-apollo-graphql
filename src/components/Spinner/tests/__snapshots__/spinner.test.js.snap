// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component has correct default styles 1`] = `
.c0 {
  height: 100%;
  text-align: center;
}

.c0 > div {
  width: 12px;
  height: 12px;
  background-color: #5d9cec;
  border-radius: 100%;
  display: inline-block;
  -webkit-animation: sk-bouncedelay 1.4s infinite ease-in-out both;
  animation: sk-bouncedelay 1.4s infinite ease-in-out both;
}

.c0 div:nth-child(1) {
  -webkit-animation-delay: -0.32s;
  animation-delay: -0.32s;
}

.c0 div:nth-child(2) {
  -webkit-animation-delay: -0.16s;
  animation-delay: -0.16s;
}

<div
  className="c0"
/>
`;
